### DISCUSSION TITLE
Sync duration of image to length of corresponding audio file?

### DISCUSSION THREAD

**Discussion by Artybartfast** [CommandPost/FCPCafe#353] [Final Cut Pro Questions]
I have a deck of slides exported in FCPx to jpegs as well as an audio file for each slide ( care of AI, whisper and 11labs). So two sets of audio files 1-10 and slides 1-10. Because I'm lazy I want to  match slide duration to audio length and of course one can do this manually and works well in FCPX but I have 20 sets of these slides so why not use type of automation!  Any suggestions and am willing to pay $ for a plug-in of some XML tweaking program. I know folks must do this all the time but now with these crazy good AI voices this is going to be very common workflow. The client want to take it into Camtasia -- and work on it there - help! I swear I searched the YouTubes and AG's interwebs and found jack shite.

**Comment by latenitefilms** [comment]
This is something we could potentially create as a CommandPost Toolbox or standalone app. How would you see it working exactly? Would you just drag in a bunch of stills and audio files that have matching names?

**Reply by Artybartfast** [reply]
I think the files jpeg and corresponding audio file  could just be linked to each other via a number. A new snag now is that if you just retime the slide /image to the length of the audio  the there is no break between the audio so we'd need the audio to start after about three seconds or so and  no audio at the end for a few seconds. This time could be the same of all slides.  I recall in the good old FCP days being about to put stuff in a spread sheet and add the timing or smoothing like that. I think with more folks using AI voices in education this would be a great little tool. Not sure about payment, I know it seems like it would be  useful. A normal editor would just edit them manually but I was curious if there was a tool to do this. Perhaps use a spreadsheet with jpeg, audio file, delay start and delay stop and just export that to CSV into a little script?
